src/vcpkg/commands.add-version.cpp

```diff
@@ -226,22 +314,27 @@ namespace
                 {
                     if (print_success)
                     {
-                        vcpkg::printf(Color::success,
-                                      "Version `%s` is already in `%s`\n",
-                                      port_version.version,
-                                      version_db_file_path);
+                        msg::println(Color::success,
+                                     msgAddVersionVersionAlreadyInFile,
+                                     msg::version = port_version.version.to_string(),
```

Comment:
This shouldn't need `.to_string()` given @BillyONeal's other PR -- otherwise we need to make the version type understood by fmt

Reply:
yeah, I believe this was before Billy's PR got merged